src/spec-node/devContainersSpecCLI.ts

```diff
@@ -328,18 +328,24 @@ async function doBuild({
 			throw new ContainerError({ description: `Dev container config (${uriToFsPath(configFile || getDefaultDevContainerConfigPath(cliHost, workspace!.configFolderPath), cliHost.platform)}) not found.` });
 		}
 		const { config } = configs;
-		let imageNameResult = '';
+		let imageNameResult = [''];
```

Comment:
Use `let imageNameResult: string[];` that will let the compiler check all code paths assign to it before it is accessed.

Reply:
That's neat. Modified, thanks